src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -49,14 +49,21 @@ GENERIC_TRUSTSTORE=${WORKDIR}/ssl-stores/generic-server-truststore.jks
 
 CA_CERT_CONFIG=${WORKDIR}/ca_cert.conf
 ROOT_CA_CERT=${WORKDIR}/ssl-stores/ca-cert
+CA_KEY=${WORKDIR}/ca-key
```

Comment:
We should remove this.

Originally when we wrote this container we debated about whether we would have a variable for the _ca-key_.

We decided against defining a variable for it because the key is only used as part of the process. It is never used as an external resource. For example:
* The keystore and truststore that are generated are used by Cassandra, tence they can be used as an external resource.
* The ca-certificate can be imported into the browser, hence it is an external resource.

The ca-key is never used externally, hence we kept it private and never bothered to make an environment variable for it.

The reason the container is set up like this is because, it is designed to serve purely as an example.